src/slider.scss

```diff
@@ -0,0 +1,132 @@
+@import "./new-settings";
+@import "./mixins";
+
+$block: #{$fd-namespace}-slider;
+
+.#{$block} {
+  @include fd-reset();
+
+  @include fd-disabled() {
+    cursor: not-allowed;
+  }
+
+  min-width: 4rem;
+  max-width: 100%;
+  padding: 1rem 0;
+  position: relative;
+
+  &__wrapper {
+    @include fd-reset();
+
+    position: relative;
+  }
+
+  &__track {
+    @include fd-reset();
+
+    height: var(--fdSlider_Track_Height);
+    width: 100%;
+    background-color: var(--fdSlider_Track_Background);
+    border-radius: var(--fdSlider_Track_BorderRadius);
+    position: relative;
+    z-index: 1;
+
+    &-range {
```

Comment:
this block can't be here in the way you use it. If it's an element (like you used it) it should be outside of the `&__track` block since it's a different element.
If you want to keep it here you need to make it a modifier class and rename it to `--range`. In this case you will have to use both classes on your div: 
```
<div class="fd-slider__track fd-slider__track--range"></div>
```

In this case inside the `&--track` block you don't need the reset, the position: relative, border-radius, etc., as they will come from `fd-slider__track ` class. That's why you make a modifier class, to modify stuff from the base class. Which approach you will choose is up to you :)